contrib/cirrus/setup.sh

```diff
@@ -8,18 +8,30 @@ show_env_vars
 
 cd $GOSRC
 
+export BuildRequires="golang go-md2man gpgme-devel device-mapper-devel btrfs-progs-devel libassuan-devel libseccomp-devel  glib2-devel  ostree-devel  make bats"
+
 case "$OS_REL_VER" in
     fedora-30)
         echo "Setting up $OS_RELEASE_ID $OS_RELEASE_VER"  # STUB: Add VM setup instructions here
+	dnf -y update
+	dnf -y install podman fuse-overlayfs $BuildRequires
         ;;
     fedora-29)
         echo "Setting up $OS_RELEASE_ID $OS_RELEASE_VER"  # STUB: Add VM setup instructions here
+	dnf -y update
+	dnf -y install podman fuse-overlayfs $BuildRequires
         ;;
     ubuntu-19)
         echo "Setting up $OS_RELEASE_ID $OS_RELEASE_VER"  # STUB: Add VM setup instructions here
+	apt-get -qq update
+	apt-get -qq install realpath $BuildRequires
+        install_fuse_overlay_from_git
         ;;
     ubuntu-18)
         echo "Setting up $OS_RELEASE_ID $OS_RELEASE_VER"  # STUB: Add VM setup instructions here
+	apt-get -qq update
+	apt-get -qq install realpath $BuildRequires
```

Comment:
apt-get is horrible to try and use under automation.  Other processes on the system will grab it's locks, and there's no provision to wait on them.  I'll scare up a retry/timeout loop I used in libpod for this...

Reply:
Okay, #349 should help with this.